man/man1/gitsh.1

```diff
@@ -239,13 +239,10 @@ branch.
 .El
 .El
 .Pp
-Unset variables will be ignored, as then are in
-.Xr sh 1 .
-For example, the commit message produced by the following command will be
-.Ic "a commit" :
-.Bd -literal -offset indent
-@ commit -m $unset "a commit"
-.Ed
+Attempting to use an unset variable will cause a command to fail. This is
+different to
```

Comment:
I had to look this up: `different to` is en-GB and `different than` is en-US. Now we know.